modules/install-sno-generating-the-discovery-iso-with-the-assisted-installer.adoc

```diff
@@ -4,7 +4,7 @@
 
 :_content-type: PROCEDURE
 [id="install-sno-generating-the-discovery-iso-with-the-assisted-installer_{context}"]
-= Generating the discovery ISO with the Assisted Installer
+= Generating the installation ISO with the Assisted Installer
 
 Installing {product-title} on a single node requires a discovery ISO, which the Assisted Installer (AI) can generate with the cluster name, base domain, Secure Shell (SSH) public key, and pull secret.
```

Comment:
The instructions below this line are overly verbose and completely outdated. Button names change, titles change, and all of that happens completely independent of OCP releases - I think they should be removed - the wizard is very friendly and clear enough as it is, the user doesn't need a manual for this. 

Pretty much all we should do is remind them to tick the "Install single node OpenShift (SNO)" checkbox

Reply:
There's also a section called "Monitoring the installation with the Assisted Installer" which should actually be "Start installation using the assisted installer" or something similar

It uses the term "Administration host" while originally it was "administration node" which is inconsistent

It doesn't mention that the user should wait for the host they're trying to discover using the discovery ISO to appear in the list of discovered hosts before proceeding with all those actions

Reply:
Also "Monitoring the installation manually" - it's not monitoring, it's installing.

Reply:
>Set the networkType. OVN-Kubernetes is the default networking for SNO. OpenShiftSDN is deprecated.

@romfreiman is it "deprecated" or completely disallowed?

Reply:
Yea. It was removed as a valid option from 4.11.